mitype/commandline.py

```diff
@@ -155,3 +170,42 @@ def load_based_on_difficulty(difficulty_level=random.randrange(1, 6)):
 
     print("Select a difficulty level in range [1,5]")
     sys.exit(2)
+
+
+def show_history(N):
+    N = int(N)
+    history_file = ".mitype_history.csv"
+    history_path = os.path.join(os.path.expanduser("~"), history_file)
+
+    if N >= -1:
+        if os.path.exists(history_path):
+            with open(history_path, "r") as file:
+                history_reader = csv.reader(file)
+                next(history_reader)
+
+                data = list(history_reader)
+                no_of_records = len(data)
+
+                k = no_of_records if N == -1 or N >= no_of_records else N
```

Comment:
instead of `N == -1` use `N <0` to cover all negative numbers.

Reply:
If I do that then the value of k will be the total number of records for all negative values.
i.e If the command is ```mitype -H -10``` it will print the default and print all the records. 😐

Reply:
Oh correct, that slipped out of my mind. Don't modify, the current behaviour is fine.